.pre-commit-config.yaml

```
@@ -31,7 +31,7 @@ repos:
 
   # Handling unwanted unicode characters
   - repo: https://github.com/sirosen/texthooks
-    rev: "0.5.0"
+    rev: "0.4.0"
```

That seems like an unintentional change that's still leftover from the merge with main
```suggestion
    rev: "0.5.0"
```